katsdpimager/test/test_fast_math.py

```diff
@@ -0,0 +1,50 @@
+"""Tests for :mod:`katsdpimager.fast_math`."""
+
+from nose.tools import assert_equal
+import numpy as np
+
+from .. import fast_math
+
+
+class TestExpj2pi:
+    def setup(self):
+        gen = np.random.Generator(np.random.PCG64(42))
+        self.a = gen.random(100) * 1000.0 - 500.0
+        self.expected = np.exp(2j * np.pi * self.a)
+
+    def test_basic(self):
+        actual = fast_math.expj2pi(self.a)
+        np.testing.assert_allclose(self.expected, actual, rtol=1e-12)
```

Comment:
The signature of `assert_allclose` has the two inputs the other way around, which might help understanding any failures:
```Python
def assert_allclose(actual, desired, rtol=1e-7, atol=0, equal_nan=True, err_msg='', verbose=True):
```

Reply:
Thanks, will fix.

Reply:
Done.